tokio/src/stream/mod.rs

```diff
@@ -303,7 +303,7 @@ pub trait StreamExt: Stream {
     /// As values of this stream are made available, the provided function will
     /// be run on them. If the predicate `f` resolves to
     /// [`Some(item)`](Some) then the stream will yield the value `item`, but if
-    /// it resolves to [`None`] then the next value will be produced.
+    /// it resolves to [`None`], then the value value will be skipped.
```

Comment:
> then the **value** value will be skipped.

I think a new typo appeared here, unless `filter_map` skips two values on a single `None` nowadays. :wink:

Reply:
Right... Duplicated word.

Reply:
I'm fixing this in #2520.